programs/vote/src/vote_state/mod.rs

```diff
@@ -557,6 +558,19 @@ pub fn process_new_vote_state(
         return Err(VoteError::TooManyVotes);
     }
 
+    if let Some(feature_set) = feature_set {
+        if feature_set.is_active(&feature_set::disallow_partial_towers::id())
+            && new_state
+                .back()
+                .expect("proposed state cannot be empty")
+                .confirmation_count()
+                != 1
+        {
+            // Top of new vote state must have confirmation 1.
+            return Err(VoteError::PartialTower);
```

Comment:
Can take this further and check adjacent pairs as well, e.g. if slot x and slot x + 1 are adjacent in the tower, their confirmation counts must be consecutive. 

Believe it's something like log_2(d - 2) difference in conf counts for adjacent slots with slot # difference d > 4, (needs to vote on enough intermediary slots and let them expire).  

But not entirely sure if we need to actually stop this weird type of behavior, once again a case of artificially increasing their lockout.